hazelcast/src/main/java/com/hazelcast/internal/management/request/ChangeWanStateRequest.java

```diff
@@ -87,4 +87,17 @@ public void fromJson(JsonObject json) {
         publisherName = getString(json, "publisherName");
         start = getBoolean(json, "start");
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
You need this method just for code coverage, right? Since equals methods have a lot of branches, this might lower the code coverage again, if you don't cover all of them.

It could be easier to create simple getter methods for those few fields and check them via asserts in the test class, e.g.:
```
assertEquals(changeWanStateRequest1.getSchema(), changeWanStateRequest2.getSchema());
assertEquals(changeWanStateRequest1.getPublisher(), changeWanStateRequest2.getPublisher());
```